### ISSUE TITLE
Unblock particular restricted/avoid roads

### ISSUE THREAD

**Issue by RicoZ-OSM** [osmandapp/Osmand#1695]
Sometimes I would need to unblock a few roads which are restricted but still valid for me.

In a similar vein, "avoids" like avoid toll,ferry etc could be unblocked on a case by case basis

**Comment by vshcherb** [comment]
Valid point, though the next request will be to change priority on the fly. It is reasonable, but will it be easier to use another profile or routing setting  or even may be to change OSM data. 
I'm afraid it is quite specific and introduces complication.

**Comment by RicoZ-OSM** [comment]
assume OSM data is correct and I have an exception of some kind as local resident. How would that work? I know how to drive around on my end but before I leave the restricted aree the overview of the route calculated by Osmand is completely worthless.

You are reading my thoughts, it would be indeed nice to change the priority of some roads. In some case it can  be done with maxspped:practical. But in cases of purely personal priority it would be nice to have that.
 
Is there any way I could specify a particular way-id in routing.xml? Or maybe it won't be difficult to have a script that manipulates downloaded maps to my personal liking?

**Comment by vshcherb** [comment]
For your request I would say it would be easier to generate your personal map which is not difficult using command line interface in that case you have full flexibility. I believe it could be part of big story to personalise routing by prioritising and reprioritising routes, but it is not planned for the nearest future.

**Comment by RicoZ-OSM** [comment]
Would it work to generate only a map of the modified routes and use it in combination with the normal map?

**Comment by vshcherb** [comment]
It should but there are known issues with it. We work on this support for Live Updates which have similar request.
